buyer-api/src/queues/dataResponseQueue.js

```diff
@@ -0,0 +1,38 @@
+import { createQueue } from './createQueue';
+import { enqueueTransaction } from '.';
+import { onBuyData, closeDataResponse } from '../facades';
+
+const createDataResponseQueue = () => {
+  const queue = createQueue('DataResponseQueue');
+
+  queue.process('buyData', async (
+    { data: { orderAddress, sellerAddress } },
+  ) => {
+    await onBuyData(orderAddress, sellerAddress, enqueueTransaction);
+  });
+
+  queue.process('closeDataResponse', async (
+    { data: { orderAddress, sellerAddress } },
+  ) => {
+    await closeDataResponse(orderAddress, sellerAddress, enqueueTransaction);
+  });
+
+  return queue;
+};
+
+const dataResponseQueue = createDataResponseQueue();
+const enqueueCloseDataResponse = (orderAddress, sellerAddress, options = {}) => {
+  const {
+    priority = 1000, attempts = 20, backoffType = 'linear',
```

Comment:
Priority values range from 1 (highest priority) to MAX_INT  (lowest priority)
Is there a reason we use 10, 100, 1000 instead of 1, 2, 3.
Maybe we could have something like:
```js
const priority = {
    HIGH: 1,
    MEDIUM: 2,
    LOW: 3,
    LOWEST: 4,
};
```
In queues/index.js or somewhere and use it like `priority.MEDIUM` so it's more self explanatory.

Reply:
It's a great idea, I will come back to this un further PRs and implement something very similar when we have all transactions updated to this new scheme.